orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/SavePipelineTask.java

```diff
@@ -38,6 +39,7 @@
 import java.util.concurrent.TimeUnit;
 
 @Component
+@ConditionalOnProperty("front50.enabled")
```

Comment:
This was a breaking change, @ezimanyi can you update to default true?

Reply:
Really sorry about that, looking now...I was trying to fix a breaking change in https://github.com/spinnaker/orca/pull/2715 but obviously did not fix it properly.